happybase/connection.py

```diff
@@ -9,11 +9,14 @@
 from thrift.transport.TSocket import TSocket
 from thrift.transport.TTransport import TBufferedTransport, TFramedTransport
 from thrift.protocol import TBinaryProtocol, TCompactProtocol
+import sasl
```

Comment:
This seems to install a C extension. Which system dependencies does that one have? Will it compile on any system?